questionnaire/utils/isQuestionnaireInstantiated/index.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+function isQuestionnaireInstantiated(sessionData) {
+    const questionnaireId = sessionData?.questionnaireId;
+    return questionnaireId || false;
```

Comment:
I'm not sure returning the `questionnaireId` or `false` is very intuitive, nor good practice? The function name also strongly implies you're getting a boolean back.

Reply:
This is a todo on my list. Will implement a `isQuestionnaireInstantiated`, and `getQuestionnaireIdFromSession` that makes more intuitive sense. Refactor is defo needed here.